docs/questions/visualizations/progress-bar.md

```diff
@@ -18,23 +18,87 @@ Progress bars give you an option to set up an alert whenever the result of a que
 
 ## How to create a progress bar
 
-To create a progress bar you'll need:
+### Data shape for the progress bar
```

Comment:
Do we need this second heading?

<img width="647" height="164" alt="image" src="https://github.com/user-attachments/assets/df3c0d5e-6ba9-4539-950b-13382776b8f6" />